sklearn_fail.ipynb

```diff
@@ -88,7 +88,10 @@
     "\n",
     "The second issue is that the hyperparameter search space is not specified for the auto-sklearn model to tune over. This may not be an issue, as the auto-sklearn documentation states that a large search space is used by\n",
     "default. However, it is better and more computationally efficient to use ranges in your search space that are known to produce good results. By simply specifying the resampling method, the model if only marginally shows\n",
-    "better performance than its default counter part. \n"
+    "better performance than its default counter part. \n",
+    "\n",
+    "While these both contribute to poor performance, from my experience, the factor that caused the auto-sklearn model to perform worse than its default counterpart is the lack of defintion of the hyperparameter search\n",
```

Comment:
If you don't define the HPO space, it simply uses the default one in auto-sklearn.